app/src/main/java/org/p2p/wallet/smsinput/SmsInputContract.kt

```diff
@@ -19,6 +19,7 @@ interface SmsInputContract {
         fun renderButtonLoading(isLoading: Boolean)
 
         fun navigateNext()
+        // not needed in other fragments, need to remove and create good reusable abstraction
```

Comment:
I'd like you to create a separate fragment and use only .xml.

Reply:
we have new design for a separate screen when the input is blocked, it appeared after i've made a task related to sms verification refactoring
![image](https://github.com/p2p-org/key-app-android/assets/130352737/d4f50579-f469-485d-b0b0-9330710d5163)

Reply:
I suggest redoing sms verification screen again, i didn't find it reusable, a lot of code that is inherited from BaseSmsInputFragment that is not needed.
I would rather exract all UI logic to a widget and use it on fragments needed

Reply:
im not doing this in this task ofc

Reply:
This screen is ready #1798